src/wp-includes/class-wp-script-modules.php

```diff
@@ -30,6 +30,17 @@ class WP_Script_Modules {
 	 */
 	private $enqueued_before_registered = array();
 
+	/**
+	 * Tracks wehther the @wordpress/a11y script module is available.
+	 *
+	 * Some additional HTML is required on the page for the module to work. Track
+	 * whether it's available to print at the appropriate time.
+	 *
+	 * @since 6.7.0
+	 * @var bool
+	 */
+	private $a11y_available = false;
```

Comment:
I don't love this, but as a private property I'm not too worried. I wanted to avoid iterating over the enqueued modules and the dependency graph repeatedly and this property allowed for pre-existing iteration to be used.

Perhaps when https://core.trac.wordpress.org/ticket/60597 is addressed a method like `::is_available( '@wordpress/a11y' )` could be implemented to know whether the module will be "available" (enqueued or in the import map).